onnxruntime/core/providers/cpu/cpu_provider_shared.cc

```diff
@@ -38,16 +38,19 @@
 #endif
 #endif
 
-#ifdef ENABLE_TRAINING
+#ifdef ENABLE_TRAINING_OPS
 #include "orttraining/training_ops/cpu/controlflow/group.h"
-#include "orttraining/training_ops/cpu/controlflow/record.h"
-#include "orttraining/training_ops/cpu/controlflow/wait.h"
-#include "orttraining/training_ops/cpu/controlflow/yield.h"
 #include "orttraining/training_ops/cpu/loss/softmax_cross_entropy_loss.h"
 #include "orttraining/training_ops/cpu/tensor/split.h"
 #include "orttraining/training_ops/cpu/optimizer/adamw/adamwbase.h"
 #endif
 
+#ifdef ENABLE_TRAINING
```

Comment:
I beleive there is some reason we do those 3 file differently?

Reply:
This distinction is preexisting : https://github.com/microsoft/onnxruntime/blob/main/orttraining/orttraining/training_ops/cpu/cpu_training_kernels.cc#L93

Reply:
AAAAHHHH looks like the treatment is different for cpu and cuda kernels: https://github.com/microsoft/onnxruntime/blob/main/orttraining/orttraining/training_ops/cuda/cuda_training_kernels.cc#L453 :(

I will fix this in my PR